debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -170,6 +172,37 @@
             .withImportance(Importance.HIGH)
             .withDescription("A token to replace on snapshot predicate template");
 
+    public static final Field RECORD_LOG_MINING_HISTORY = Field.create("log.mining.record.history")
+            .withDisplayName("Record Log Mining history")
+            .withType(Type.BOOLEAN)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.HIGH)
+            .withDefault("false")
+            .withDescription("Flag to record Log Mining history");
+
+    public static final Field LOG_MINING_HISTORY_RETENTION = Field.create("database.history.retention.hours")
+            .withDisplayName("Log Mining history retention")
+            .withType(Type.STRING)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.MEDIUM)
+            .withDefault("4")
+            .withDescription("Hours to keep Log Mining history");
+
+    public static final Field RAC_SYSTEM = Field.create("database.rac")
+            .withDisplayName("Oracle RAC")
+            .withType(Type.BOOLEAN)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.HIGH)
+            .withDefault("false")
+            .withDescription("Flag to if it is RAC system");
+
+    public static final Field RAC_NODE_NAMES = Field.create("database.rac.nodes.ip")
```

Comment:
host names did not work for me, IP worked. It could be a consequence or not.
We had no time to re-check. Please consider it as IPs

Reply:
I've indicated the field to be either host names or IP addresses.  Both should work in theory since the field is meant to be a comma-separated list of values.  One thing to note for you downstream is that we've done some consolidation of configuration options introduced by your code. 

For Oracle RAC, we've removed the boolean toggle in favor of just determining whether Oracle RAC is enabled based on if the comma-separated value-set for `rac.nodes` is empty or not.  Additionally, we did something similar with history recording where we determine whether its enabled based on if the retention for history recording is greater than `0`.